Stripe/STPPaymentMethodsViewController.m

```diff
@@ -190,6 +203,14 @@ - (void)internalViewControllerDidSelectPaymentMethod:(id<STPPaymentMethod>)payme
     [self finishWithPaymentMethod:paymentMethod];
 }
 
+- (void)internalViewControllerDidDeletePaymentMethod:(id<STPPaymentMethod>)paymentMethod {
+    if ([self.delegate isKindOfClass:[STPPaymentContext class]]) {
+        // Notify payment context to update its copy of payment methods
+        STPPaymentContext *paymentContext = (STPPaymentContext *)self.delegate;
+        [paymentContext removePaymentMethod:paymentMethod];
+    }
```

Comment:
this is a little bit of a workaround to avoid adding the "did delete payment method" to the STPPaymentMethodsViewController delegate API